its/ruling/src/test/resources/autoscan/autoscan-diff-by-rules.json

```diff
@@ -2425,7 +2425,7 @@
   },
   {
     "ruleKey": "S5852",
-    "truePositives": 73,
+    "truePositives": 75,
```

Comment:
To be honest, I'm really unsure how it is possible to have a different result here, without modifying the test code...